tests/test_api.py

```diff
@@ -21,3 +24,22 @@ def test_geojson():
 
     osm_geo = api.get("node(area:3602758138)[amenity=cafe]")
     assert len(osm_geo["features"]) > 1
+
+
+def test_geojson_extended():
+
+    class API(overpass.API):
+        def _get_from_overpass(self, query):
+            return pickle.load(open(os.path.join(os.path.dirname(__file__), "example.response"), "rb"))
+
+    # api = overpass.API()
+    # osm_geo = api.get("rel(6518385);out body geom;way(10322303);out body geom;node(4927326183);", verbosity='body geom')
+    # pickle.dump(api._get_from_overpass("[out:json];rel(6518385);out body geom;way(10322303);out body geom;node(4927326183);out body geom;"),
+    #             open(os.path.join(os.path.dirname(__file__), "example.response"), "wb"),
+    #             protocol=2)
+    # geojson.dump(osm_geo, open(os.path.join(os.path.dirname(__file__), "example.json"), "w"))
```

Comment:
Is there a specific reason why you left this commented code here?

Reply:
Basically the commented code is needed to create the _example.response_ and _example.json_ files. The _example.response_ file is subsequently used to fake the _get_from_overpass method and the _example.json_ file is the reference that we are asserting against. 
It's not really safe to make calls to the actual API in the test as the underlying data can change at any moment.

Reply:
Alright makes sense. Could convert it to an explaining "example" comment? Then everybody know how this is handled in the future.

Reply:
Done!